src/theme/material/select.m.css

```diff
@@ -35,6 +45,11 @@
 .value {
 	composes: mdc-select__selected-text mdc-line-ripple from '@material/select/dist/mdc.select.css';
 	text-align: left;
+	color: var(--mdc-text-color) !important;
```

Comment:
you'll need to add a disabled colour too

Reply:
Fixed!

![image](https://user-images.githubusercontent.com/2008858/79985282-5b3eb680-8478-11ea-9afd-2a7124256bc1.png)